src/index.js

```diff
@@ -45,6 +47,8 @@ export {
   OpacityField,
   RadiusField,
   WidthField,
+  LineCapField,
+  LineJoinField,
```

Comment:
The `OffsetField` is missing in the exports. Is this intended?

Reply:
No, it is not. Thanks for noticing. Since `OffsetField` already existed before, I did not check if it will be exported. But I will change this.